test/Subdomain.spec.js

```diff
@@ -2217,7 +2217,7 @@ describe('DCL Names V2', function() {
       expect(subdomainOwner).to.be.equal(anotherUser)
     })
 
-    it('reverts when trying to change the resolver by an unauthorized account', async function() {
+    it('reverts when trying to change the resolver by an unauthorized account [ @skip-on-coverage ]', async function() {
```

Comment:
As noted in a [comment][1] in the parent issue at solidity-coverage, these tests fail with InvalidJump instead of revert under coverage. 

Tbh I'm not sure if the coverage report is actually affected by skipping these - I had difficulty locating the implementation for the relevant methods in the code here. 

[1]: https://github.com/sc-forks/solidity-coverage/issues/467#issuecomment-574344005